core/kazoo_services/src/kazoo_services_maintenance.erl

```diff
@@ -149,7 +149,11 @@ js_quantify_map() ->
        "        case 'user':"
        "            emit(['users', doc.priv_level], 1);"
        "            if (doc.qubicle && doc.qubicle.enabled) {"
-       "                emit(['qubicle', 'recipients'], 1);"
+       "                if (doc.qubicle.pvt_prod_level == 'pro') {"
+       "                    emit(['qubicle', 'pro_recipients'], 1);"
```

Comment:
@mark2600 while this is fine, I would prefer if we user qubicle_offering for both emitted paths.  Going with that, we could simplify this to simply:

emit(['qubicle', doc.pvt_qubicle_offering || 'basic' + '_recipients'], 1);